### PR TITLE
Open up a text box when giving feedback from emoji

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: static/js/components/nl_feedback.tsx [hunk 1]
```diff
@@ -62,12 +62,93 @@ const FEEDBACK_OPTIONS = [
...
+        <div className="backdrop" onClick={handlePopUpClose}></div>
+      )}
+
+      {/* Pop-up text box */}
+      {isPopUpVisible && (
+        <div className="popup-box">
+          <textarea
+            value={textInput}
+            onChange={handleInputChange}
+            placeholder="Enter your feedback here..."
```
Comment (pradh) [inline static/js/components/nl_feedback.tsx:128]: super nit:  drop `...`

File: static/js/components/nl_feedback.tsx [hunk 2]
```diff
@@ -28,13 +28,13 @@ import {
   getNlChartId,
 } from "../utils/nl_interface_utils";
 
-interface NlChartFeedbackPropType {
-  // This is a fixed ID format string
-  // e.g., "pg0_cat_1_blk_2_col_3_tile_4"
-  id: string;
```
Comment (pradh) [inline static/js/components/nl_feedback.tsx:34]: nit: Keep this comment for reference in EmojiPropType?

**Review by pradh — APPROVED** [review]
Thanks Bo!

File: static/js/components/nl_feedback.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/components/nl_feedback.tsx]: Done

File: static/js/components/nl_feedback.tsx (hunk 2 above)
Comment (shifucun) [reply static/js/components/nl_feedback.tsx:34]: This is moved to below

**Pull request merged**